tests/copy_file/BUILD

```diff
@@ -0,0 +1,86 @@
+load("//rules:copy_file.bzl", "copy_file")
```

Comment:
(same comment as your other PR: I find the overall goal and structure of this BUILD file unclear. Can you add a top-level comment describing it?)

Reply:
Done.